src/desdemona/tasks/kafka.clj

```diff
@@ -2,6 +2,12 @@
   (:require [taoensso.timbre :refer [info]]
             [cheshire.core :as json]))
 
+(defn deserialize-message-json [bytes]
+  (try
+    (json/parse-string (String. bytes "UTF-8"))
+    (catch Exception e
+      {:error e})))
```

Comment:
Correct. This code is re-stolen* from the template, so it's not clear what the Onyx best practice is.

_* It came from the template but we weren't using it until now, so it got cleaned up. Now it's back._